cookbook_artifacts_test.go

```diff
@@ -0,0 +1,134 @@
+package chef
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const CBAListResponseFile = "test/cookbook_artifacts_list_response.json"
+const CBAGetResponseFile = "test/cookbook_artifacts_get_response.json"
+const CBAGetVersionResponseFile = "test/cookbook_artifacts_getversion_response.json"
+
+func TestListCBA(t *testing.T) {
+	setup()
+	defer teardown()
+
+	file, err := ioutil.ReadFile(CBAListResponseFile)
+	if err != nil {
+		t.Error(err)
+	}
+
+	mux.HandleFunc("/cookbook_artifacts", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, string(file))
+	})
+
+	data, err := client.CookbookArtifacts.List()
+	if err != nil {
+		t.Error(err)
+	}
+
+	if data == nil {
+		t.Fatal("We should have some data")
+	}
+
+	if len(data) != 2 {
+		t.Error("Mismatch in expected policies count. Expected 2, Got: ", len(data))
```

Comment:
nit: expected cookbook artifacts, not policies

Reply:
Fixed. Thanks!